userdocs/src/usage/eks-managed-nodes.md

```diff
@@ -195,6 +194,14 @@ Note that `maxUnavailable` cannot be higher than `maxSize`. Also, `maxUnavailabl
 
 This feature is only available for managed nodes.
 
+## Updating managed nodegroups
+It is also possible to update specific fields of a managed nodegroup with the command `eksctl update nodegroup` without using `upgrade`. 
+
+This new command currently only supports a few fields. It is intended to be used as a way to modify the configuration of a nodegroup without triggering an entire upgrade. 
```

Comment:
its none in this PR right? and then the first field is introduced in the follow PR? Might be worth moving the docs to the follow up PR

Reply:
do you mean that we should move all user docs about `update` to the follow-up PR? the other PR is pretty much ready to go right after this one and they're meant to be released together so I don't think it makes much difference tbh, what do you think?

Reply:
I think in that case lets leave it, but for the future If I was to go to this doc for a git blame, I wouldn't find the PR that actually introduced this functionality which would be slightly confusing, although we link to that PR in this PR so its not that bad :smile:

Reply:
ok I'll point to the other PR in this PR's merge commit!